ethereum/executionlayer/src/main/java/tech/pegasys/teku/ethereum/executionlayer/ExecutionLayerManagerImpl.java

```diff
@@ -260,6 +281,24 @@ && isBuilderAvailable()
         && spec.atSlot(slot).getMilestone().isGreaterThanOrEqualTo(SpecMilestone.BELLATRIX)) {
       LOG.warn("Builder endpoint is available but a non-blinded block has been requested");
     }
+    if (spec.atSlot(slot).getMilestone().isGreaterThanOrEqualTo(SpecMilestone.CAPELLA)) {
+      return executionEngineClient
+          .getPayloadV2(executionPayloadContext.getPayloadId())
+          .thenApply(ExecutionLayerManagerImpl::unwrapExecutionClientResponseOrThrow)
+          .thenCombine(
+              SafeFuture.of(
+                  () ->
+                      SchemaDefinitionsCapella.required(spec.atSlot(slot).getSchemaDefinitions())
+                          .getExecutionPayloadSchema()),
```

Comment:
I think this would just be `SchemaDefinitiosnBellatrix.required` - it will then work for either Bellatrix or Capella.

Reply:
i guess i'm up to 'if capella is in play, i'll call v2 mode' currently, but I see what you're saying...